openmrs-19/src/main/java/org/motechproject/openmrs19/resource/PatientResource.java

```diff
@@ -39,12 +39,21 @@
     /**
      * Returns the UUID of the MOTECH patient identifier.
      *
-     * @return  the UUD of the MOTECH patient identifier
+     * @return  the UUID of the MOTECH patient identifier
      * @throws HttpException  when there were problems while fetching MOTECH patient identifier
      */
     String getMotechPatientIdentifierUuid() throws HttpException;
 
     /**
+     * Returns the patient identifier name for the given uuid.
```

Comment:
This seems to perform various other operations than just returning patient identifier name. Please correct the javadoc to include everything that happens after calling this method and preferably rename this method, as it's not just a "get".